frontend/src/components/Form.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, useState } from "react";
+import { withRouter, history } from "react-router";
+
+// import {onSubmit} from './App';
+
+// eslint-disable-next-line
+function Form(props) {
+  const [value, setValue] = useState("");
+
+  return (
+    <div className="searchForm">
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          props.onSubmit(value);
+        }}
+      >
+        <input
+          type="text"
+          id="filter"
+          placeholder="Full name or username"
+          value={value}
+          onChange={e => {
+            e.preventDefault();
+            setValue(e.target.value);
+          }}
+        />
+        <button>Submit</button>
+      </form>
+    </div>
+  );
+}
+
+export default Form;
+
+// class Bla extends Component{
```

Comment:
Might can be deleted?

Reply:
yes, but I want to leave this in for learning purposes at the moment to understand better the difference between class and functional components